src/mmw/js/src/modeling/models.js

```diff
@@ -625,11 +631,14 @@ var ScenarioModel = Backbone.Model.extend({
         var self = this,
             results = this.get('results'),
             taskModel = this.get('taskModel'),
+            filteredModifications = this.get('modifications').filter(function(mod) {
```

Comment:
Consider renaming to `nonZeroModifications` or `modificationsInsideAoI` or something more descriptive of intent.